lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -242,8 +242,8 @@ sub engine_workit {
             sleep 5 and update_setup_status until $rsync_request->processed;
             my $exit = $rsync_request->result;
 
-            log_info("rsync: " . $rsync_request->output, channels => 'autoinst');
-            return {error => "Failed to rsync tests: exit " . $exit} unless $exit == 0;
+            if (my $output = $rsync_request->output) { log_info("rsync: " . $output, channels => 'autoinst') }
+            return {error => "Failed to rsync tests: exit " . $exit} unless ($exit && $exit == 0);
```

Comment:
seems this broke on o3 today, see https://progress.opensuse.org/issues/46658

I think that checking for `$exit && exit == 0` is wrong because when it's `0` it should not treat it as error so it should be `defined $exit && $exit == 0`, right?

@Martchus did we miss this in our tests because of "lsof"? Or is something missing in tests? I thought we had "full-stack + asset cache + test cache" tested?